modules/tensor_mechanics/src/actions/TensorMechanicsAction.C

```diff
@@ -56,6 +56,9 @@ validParams<TensorMechanicsAction>()
   params.addParam<std::string>(
       "strain_base_name",
       "The base name used for the strain. If not provided, it will be set equal to base_name");
+  params.addParam<std::vector<TagName>>(
```

Comment:
Damnit! I didn't see this...

Reply:
It'd be nice to get this in sooner rather than later so I can start applying reference residual to your thermal contact methods.